app/models/grouping.rb

```diff
@@ -22,10 +22,6 @@ class Grouping < ActiveRecord::Base
   has_many :accepted_student_memberships,
            -> { where 'memberships.membership_status' => [StudentMembership::STATUSES[:accepted], StudentMembership::STATUSES[:inviter]] },
            class_name: 'StudentMembership'
-  has_many :accepted_students,
```

Comment:
I'm not sure why you deleted this.

Reply:
Because it has two of those, it was duplicate.